how-to/features/tls-vault.rst

```diff
@@ -0,0 +1,69 @@
+TLS Vault
+=========
+
+This feature is used to encrypt all cloud service endpoints (both public
+and private) using TLS certificates generated by Vault, which acts as an intermediary CA.
+It does this by interfacing with the existing Traefik instances in the
+cloud. A Traefik instance is associated with either public or private
+cloud traffic.
+
+Prerequisites
+-------------
+
+To use TLS Vault, you must enable the Vault feature in your cloud and unseal and authorize the Vault charm.
+This is done by following guide :doc:`Enable Vault </how-to/features/vault>`.
+
+Enable TLS Vault
+----------------
+
+To enable TLS Vault, you’ll need to provide information that identifies your
+chosen Certificate Authority. Do this by specifying a CA certificate and
+its CA certificate chain.
+
+Run the following command to enable TLS Vault for public endpoints:
+
+::
+
+   sunbeam enable tls vault --ca <base64 encoded ca certificate> --ca-chain <base64 encoded ca chain>
+
+To enable TLS Vault for public, internal and rgw endpoints, be explicit by
+using the ``--endpoint`` option:
+
+::
+
+   sunbeam enable tls vault --ca <base64 encoded ca certificate> --ca-chain <base64 encoded ca chain> --endpoint public --endpoint internal --endpoint rgw
+
+Disable TLS Vault
+-----------------
+
+To disable TLS Vault in the cloud, run the following command:
+
+::
+
+   sunbeam disable tls vault
+
+All services will work as if TLS Vault was never enabled.
```

Comment:
What does this mean? What exactly does the disable command disable?

Reply:
This removes manual-tls-certificates charm and removes Vault from being the certificate provider. It will revert back to the previous state of the sunbeam cluster (as if the user did not run the `sunbeam enable tls vault`)